src/posture_estimator.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file posture_estimator.h
+ * @brief Interface for representation of user's pose
+ *
+ * @copyright Copyright (C) 2021  Conor Begley
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "posture_estimator.h"
+
+#include <stdio.h>
+
+#include <cmath>
```

Comment:
Bit pernickety but should we have whitespace between the includes?

Reply:
@miklasr   @Ashwin-MJ Do we have a standard for this?

Reply:
I just go with the cpplint formatting. I think it's fine tbh, what do you think @miklasr

Reply:
Okay, no problem I was just making sure. :)